shared/common-adapters/box.desktop.js

```diff
@@ -24,36 +24,34 @@ const injectGaps = (component, _children, gap, gapStart, gapEnd) => {
   return children
 }
 
+const box2 = (props: Box2Props) => {
```

Comment:
i was toying with exporting this directly and using it but i think i want to leave it out for now, fine to delegate to this function in Box2 though, so lets leave that for now